packages/components/src/dropdown/index.native.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+class Dropdown extends Component {
```

Comment:
This is a very simple class that displays an inserter (button) along with a modal and manages the state (visible/hidden) of the modal. It's pretty much like its web version only it doesn't use divs and does not unmount the modal (set the isVisible props instead)